etcdctl/command/member_commands.go

```diff
@@ -86,9 +86,8 @@ func actionMemberAdd(c *cli.Context) {
 	mAPI := mustNewMembersAPI(c)
 
 	url := args[1]
-	ctx, cancel := context.WithTimeout(context.Background(), client.DefaultRequestTimeout)
+	ctx, cancel := contextWithTotalTimeout(c)
```

Comment:
I would `defer cancel()` here. It is reasonable because the ctx is for the whole function.